xml/System.Diagnostics/Activity.xml

```diff
@@ -865,9 +874,10 @@ The root ID is a substring from the <xref:System.Diagnostics.Activity.Id> or the
         <Parameter Name="propertyValue" Type="System.Object" Index="1" FrameworkAlternate="net-5.0" />
       </Parameters>
       <Docs>
-        <param name="propertyName">To be added.</param>
-        <param name="propertyValue">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="propertyName">The name to associate the value with.<see cref="P:System.Diagnostics.Activity.OperationName" /></param>
+        <param name="propertyValue">The object to attach and map to the property name.</param>
+        <summary>SetCustomProperty allow attaching any custom object to this Activity object.
+            If the property name was previously associated with other object, SetCustomProperty will update to use the new propert value instead.</summary>
         <remarks>To be added.</remarks>
```

Comment:
>To be added [](start = 17, length = 11)

In the remark section, please state that users of this API should try to use a unique propertyName to avoid conflicting with the other API callers.